scripts/install_dev_deps.sh

```diff
@@ -2,4 +2,4 @@
 
 # Merlin, utop, ocp-indent, ocamlformat, and patdiff are all for developer assistance
 opam pin -y ocamlformat 0.8
-opam install -y merlin utop ocp-indent patdiff
+opam install -y ocamlformat.0.8 merlin utop ocp-indent patdiff
```

Comment:
I think `pin`ing ocamlformat should also install it, but I'm going to merge this for the fix anyway :)